lib/byebug/helpers/toggle.rb

```diff
@@ -26,14 +24,22 @@ def enable_disable_breakpoints(is_enable, args)
           selected_breakpoints = all_breakpoints.select do |b|
             selected_ids.include?(b.id)
           end
+          return selected_breakpoints
```

Comment:
This kind stylistic stuff, just do it however you want. If you get to the stage where you have to work across several languages that work differently, perhaps you'll see things differently. 

I could just as easily imagine without this the comment "Explicit is better than implicit so put in a return".